tests/Functional/run.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
Mohl bys nějak vysvětlit, proč je tady tohle?

Reply:
Protoze chci otestovat Common, ktery tu je a pouziva abstract Extractor, ale neni tu zadny run, ktery by to s tim spustil, proto je tu tento primo pro testy.

Reply:
Jo, jasně, chápu, díky...